source/conf.py

```diff
@@ -17,7 +17,7 @@
 project = "ds-fastapi-middleware"
 copyright = "2024, skerve yuan1979"
 author = "skerve yuan1979"
-release = "2024.3.2"
+release = "2024.4.2"
```

Comment:
the conf.py and pyproject.toml use the same release, it should be in a common file.